internal/conf/config.go

```diff
@@ -5,20 +5,27 @@ import (
 	"os"
 )
 
+// Определяет параметры подключения к хранилищу
 var (
-	BndAdd   string
-	RetAdd   string
+	// BndAdd - адрес на котором будет размещаться сервер
+	BndAdd string
+	// RetAdd - адрес который будет присоединяться к результируюему ответу
+	RetAdd string
+	// FilePath - путь к файлу в котором будет храниться база с ссылками
 	FilePath string
```

Comment:
Для комментариев к полям (не очень длинных) есть другой гайдлайн, поддерживаемый всеми IDE:
```suggestion
	BndAdd string // адрес на котором будет размещаться сервер
	RetAdd string // адрес который будет присоединяться к результируюему ответу
	FilePath string // путь к файлу в котором будет храниться база с ссылками
```